process-link/src/main/kotlin/com/ritense/processlink/importer/ProcessLinkImporter.kt

```diff
@@ -68,13 +68,17 @@ class ProcessLinkImporter(
 
             val deployDto = objectMapper.treeToValue<ProcessLinkDeployDto>(node)
 
-            val processLinkDto = processLinkService.getProcessLinkMapper(deployDto.processLinkType)
+            val processLinkCreateDto = processLinkService.getProcessLinkMapper(deployDto.processLinkType)
                 .toProcessLinkCreateRequestDto(deployDto)
 
             try {
-                processLinkService.createProcessLink(processLinkDto)
+                processLinkService.createProcessLink(processLinkCreateDto)
             } catch (e: ProcessLinkExistsException) {
```

Comment:
It would be better if there was a function to check if the processlink exists. something like:
```
if (processLinkService.processLinkExists(...)) {
```